Simulations/python/scopesimWrapper.py

```diff
@@ -83,16 +83,15 @@ def simulate(fname, rcp, small=False):
 
     # keywords we always have
 
-    print(fname)
-    print(props)
     shutter = False
     cmd["!OBS.catg"] = props["catg"]
     cmd["!OBS.type"] = props["type"]
     cmd["!OBS.tech"] = props["tech"]
     cmd["!OBS.mjd-obs"] = props["MJD-OBS"]
     cmd["!OBS.dateobs"] = props["dateobs"]
     # TODO: Ensure ndfilter_name is always defined.
-    cmd["!OBS.ndfilter_name"] = props.get("ndfilter_name", "open")
+    cmd["!OBS.nd_filter_name"] = props["ndfilter_name"]
```

Comment:
I added the `get()` in #122 because it was necessary, see the TODO above the line. Did you ensure that `ndfilter_name` is always defined in the yaml files?